components/example-notebook-servers/jupyter-pytorch-full/requirements.txt

```diff
@@ -1,23 +1,23 @@
 # kubeflow packages
-kfp==1.6.3
-kfp-server-api==1.6.0
-kfserving==0.5.1
+kfp==2.3.0
+kfp-server-api==2.0.1
+kfserving==0.6.1
```

Comment:
Should we drop this package and install the corresponding kserve package instead?

Reply:
yes 0.11.1 from here https://pypi.org/project/kserve/